components/raftstore/src/store/replication_mode.rs

```diff
@@ -185,6 +195,16 @@ impl GlobalReplicationState {
             ReplicationMode::Majority => None,
         }
     }
+
+    pub fn check_store_available(&self, store_id: u64) -> bool {
+        match self.status.get_mode() {
+            ReplicationMode::Majority => true,
+            ReplicationMode::DrAutoSync => {
+                let available_stores = self.status.get_dr_auto_sync().get_available_stores();
+                available_stores.is_empty() || available_stores.contains(&store_id)
```

Comment:
Why empty is true?

Reply:
PD only set it in `wait_async` and `async` states. Other states, we do not care the store list.